### PR TITLE
job:  #9337  Adding mapping paper to publishable repo.

### PR DESCRIPTION
**Pull request by cortlandstarrett**

### CODE REVIEW

File: doc/notes/mapping/9337_mapping.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+but is very specific to the target architecture chosen technologies.
+The model adheres to
+semantics that clearly define and constrain each element in the Design.
+Each element of the system/software architecture is illustrated in a
+model of the software architecture.
+
+With a model of Requirements and a model of Design, a model-to-model (M2M)
+transformation can be done that maps these model elements together
+into the Implementation.
+As a final step of the Design process a set of rules is defined.
```
Comment (lwriemen) [inline doc/notes/mapping/9337_mapping.md]: Need comma after process.

File: doc/notes/mapping/9337_mapping.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+One rule is defined for each type of element in the Requirements.
+Each rule explains how to pair an element in the Requirements with
+an element in the Design.
+This set of mapping rules is the core of the Model Translation.
+These rules can be applied manually.
+
+With models, much of this mapping and translation can be automated.
```
Comment (lwriemen) [inline doc/notes/mapping/9337_mapping.md]: These two sentences could be combined. "These rules can be applied manually, but much of this mapping and translation of the models can be automated."

File: doc/notes/mapping/9337_mapping.md [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+
+## Implementation  
+Implementation realizes the Requirements within the constraints of the
+Design.
+
```
Comment (lwriemen) [inline doc/notes/mapping/9337_mapping.md]: Move "A successful Implementation is proven by Testing." to the end of this paragraph, unless you're trying to intimate that it _must_ be proven by testing when done ad hoc.

File: doc/notes/mapping/9337_mapping.md [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+In some systems, the process of Implementation is not clearly specified.
+In such cases the implementation team must satisfy the input Requirements
+and Design in an _ad hoc_ fashion.  A successful Implementation is proven
+by Testing.  But the Implementation team is not given a process whereby
+the Requirements and Design _translate_ into the Implementation.  Such
+a set of rules make Implementation easier, faster and more precise.
+
```
Comment (lwriemen) [inline doc/notes/mapping/9337_mapping.md]: This paragraph doesn't read well after the first two sentences. Maybe it should be:
In some systems, the process of Implementation is not clearly specified. 
In such cases the Implementation team must satisfy the input Requirements and Design in an _ad hoc_ fashion.
By giving the Implementation team a set of translation rules, the process whereby the Requirements and 
Design _translate_ into the Implementation is made easier, faster, and more precise. This set of rules is the Model Translation.

File: doc/notes/mapping/9337_mapping.md [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+Following is a simple example of Model Translation using Mapping.
+The example is a GPS Tracker used to track the location of a person
+or boat sailing on the sea.
+The example shows Requirements, Design and a Mapping to Implementation.
+
+### GPS Tracker  
+
+#### GPS Tracker:  Requirements  
+Consider a GPS Tracker that acquires and accumulates track points
+on a periodic time basis to record the path of the person or ship
```
Comment (lwriemen) [inline doc/notes/mapping/9337_mapping.md]: "track points" isn't clear, but I see why you want to introduce the term here. I'd use "location information (track points)"

File: doc/notes/mapping/9337_mapping.md (hunk 3 above)
Comment (cortlandstarrett) [inline doc/notes/mapping/9337_mapping.md]: kinda am...  I reworded it to get that point through a bit more clearly.

**Pull request merged**